person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, rentals
   attr_accessor :name, :age
 
   def initilizate(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
```

Comment:
Kindly correct the method name initializate to initialize.
However, if things have change, kindly let me know.

## :slightly_smiling_face:

Reply:
done

Reply:
Well resolved 👍🏽